src/Model/BrandInterface.php

```diff
@@ -30,4 +30,14 @@ public function getName(): ?string;
      * @param string|null $name
      */
     public function setName(?string $name): void;
+
+    /**
+     * @return \DateTime
+     */
+    public function getCreatedAt(): \DateTime;
```

Comment:
We can't change the interface without releasing a new major, so instead add these method as phpdoc method on the interface. Then we will add them to the actual interface in `v3`